src/ontology/cl-edit.owl

```diff
@@ -19054,13 +19054,20 @@ SubClassOf(obo:CL_0002451 obo:CL_0000048)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:10704434"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:11756508"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:20142420"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:2182078"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:24133281"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:9826671"^^xsd:string) obo:IAO_0000115 obo:CL_0002453 "A progenitor cell of the central nervous system that can differentiate into oligodendrocytes or type-2 astrocytes. This cell originates from multiple structures within the developing brain including the medial ganglion eminence and the lateral ganglionic eminence. These cells migrate throughout the central nervous system and persist into adulthood where they play an important role in remyelination of injured neurons."^^xsd:string)
 AnnotationAssertion(oboInOwl:created_by obo:CL_0002453 "tmeehan"^^xsd:string)
 AnnotationAssertion(oboInOwl:creation_date obo:CL_0002453 "2010-11-10T02:51:34Z"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24133281"^^xsd:string) oboInOwl:hasBroadSynonym obo:CL_0002453 "O-2A/OPC"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11756508"^^xsd:string) oboInOwl:hasBroadSynonym obo:CL_0002453 "O2A/OPC"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:2253328"^^xsd:string) oboInOwl:hasBroadSynonym obo:CL_0002453 "oligodendrocyte-type 2 astrocyte (O-2A) progenitor cell"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11756508"^^xsd:string) oboInOwl:hasBroadSynonym obo:CL_0002453 "oligodendrocyte-type-2 astrocyte progenitor cell"^^xsd:string)
+AnnotationAssertion(Annotation(<http://purl.org/dc/terms/contributor> "http://orcid.org/0000-0002-0819-0473") Annotation(oboInOwl:hasDbXref "PMID:24133281"^^xsd:string) oboInOwl:hasBroadSynonym obo:CL_0002453 "oligodendrocyte/type-2 astrocyte progenitor cell"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0002453 "cell"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10704434"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:2182078"^^xsd:string) Annotation(oboInOwl:hasDbXref "https://veteriankey.com/of-the-central-and-peripheral-nervous-system/"^^xsd:string) rdfs:comment obo:CL_0002453 "This cell type can be purified from optic nerves and other regions of the embryonic, postnatal and adult rat CNS and can be differentiated in vitro to oligodendrocytes and type-2 astrocytes. Some references use the terms 'oligodendrocyte precursor cell' and 'oligodendrocyte type-2 astrocyte (O-2A) progenitor' to refer to the same entity (e.g., PMID:10704434), while others describe an oligodendrocyte type-2 astrocyte (O-2A) progenitor cell as the precurosr to oligodendrocyte- and type-2 astrocyte- progenitor cells (e.g, https://veteriankey.com/of-the-central-and-peripheral-nervous-system/)."^^xsd:string)
```

Comment:
Figure is from https://www.us.elsevierhealth.com/essentials-of-domestic-animal-embryology-9780702028991.html - can cite it as ISBN:9780702028991 instead of website